### DISCUSSION TITLE
How do I see all commits *not* made by bot?

### DISCUSSION THREAD

**Discussion by zhangyiatmicrosoft** [maplibre/maplibre-gl-js#3795] [General]
Rookie question...seems GH web interface only allows to filter commits on user, but there is no "negate" filter?
I found myself very often need to see all commits made by human being ...

**Comment by HarelM** [comment]
Not sure about commit, but you can easily do it for PRs, I think you need to add "-" before the filter. Maybe the same can be applied to commit, IDK.

**Reply by zhangyiatmicrosoft** [reply]
cool PR works! 
<code>is:pr is:merged -author:app/dependabot </code>
